thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -692,6 +692,8 @@ class CIndexReadSlaveBase : public CSlaveActivity
                     }
                 }
             }
+            data.read(fileTableStart);
+            setupSpace4FileStats(fileTableStart, true, fileStats, super!=nullptr, super?super->querySubFiles():0);
```

Comment:
I'll probably rename that parameter.  reInit==true causes it to always use the fileStats table for the stats. If reInit==false, then it only uses the fileStats table if it's necessary (sometimes it's possible not bother using fileStats at all and just  use the stats from activity stats).

Whenever you see reInit parameter passed as true, it is to tell setupSpace4FileStats that filestats must be used (because we know multiple files are being processed by the activity.)

Reply:
I have renamed reInit parameter to statsForMultipleFiles.   The fileStats is either used for a single super file or multiple files.